### ISSUE TITLE
Fix `node_modules` conflict with ITs

### ISSUE THREAD

**Issue by victor-diez-sonarsource** [SonarSource/SonarJS#3993]
Since the move of ESlint-bridge to the root of the repo, running the integration tests requires deleting the node_modules/ folder. After that, working with the ESLint-bridge or doing a `git commit` requires to re-install the dependencies.
It would be nice to have a way to avoid doing any action, that would be resilient in case we interrupt ruling tests.

Depends on #3992

**Comment by Wohops** [comment]
Migrated to Jira. New ticket number: JS-88
